torch/distributed/pipelining/PipelineStage.py

```diff
@@ -1034,6 +1108,8 @@ def __init__(
         else:
             self.outputs = _create_empty_tensors(output_args, device)
 
+        self._configure_outputs_meta(self.outputs)
```

Comment:
While in tracer case we don't need to cast things to meta (because `node.meta["val"]` is already a fake tensor), here do we need to cast things to meta? You can put the cast logic in `_configure_outputs_meta` though.

Reply:
you're saying its ok to keep .to(meta) inside configure_outputs_meta?  i think its harmless to do it an extra time.

Reply:
Either way is fine. Up to you :) 
I was thinking FakeTensor.to("meta") is an no-op and it can be put in `_configure_outputs_meta` without harm.
But at a second thought, maybe I am badly wrong. (FakeTensor != meta tensor)